README.md

```diff
@@ -124,11 +125,11 @@ make test-rust-all
 ```
 
 The integration test suite requires `python`. By default, the integration test
-suite will create a virtualenv, build the Relay binary with processing enabled,
-and run a set of integration tests:
+suite will create a Python virtual environment, build the Relay binary with
```

Comment:
Since the tool and the binary are called `virtualenv` (see https://virtualenv.pypa.io/en/latest/), can we leave this as virtualenv? This sounds more familiar.